CONTRIBUTING.md

```diff
@@ -247,38 +236,38 @@ Languages with less than 90% coverage will be removed in a future Nightscout ver
 
 | Language      | List of translators | Status
 | ------------- | -------------------- |-------------------- |
-| Български (`bg`) |Please volunteer| OK |
-| Čeština (`cs`) |Please volunteer|OK |
+|  (`bg`) |Please volunteer| OK |
+| etina (`cs`) |Please volunteer|OK |
 | Deutsch (`de`) |[@viderehh] [@herzogmedia] |OK |
 | Dansk (`dk`)  | [@janrpn] |OK |
-| Ελληνικά `(el`)|Please volunteer|Needs attention: 68.5%|
+|  `(el`)|Please volunteer|Needs attention: 68.5%|
 | English (`en`)|Please volunteer|OK|
-| Español (`es`) |Please volunteer|OK|
+| Espaol (`es`) |Please volunteer|OK|
```

Comment:
something goes wrong here with accents. Also please use `wip/013alpha` as basis, to prevent merge errors